work/v1/types.go

```diff
@@ -66,6 +70,64 @@ type DeleteOption struct {
 	SelectivelyOrphan *SelectivelyOrphan `json:"selectivelyOrphans,omitempty"`
 }
 
+type StatusFeedbackOption struct {
+	// ResourceIdentifier represents the group, resource, name and namespace of a resoure.
+	// +kubebuilder:validation:Required
+	// +required
+	ResourceIdentifier ResourceIdentifier `json:"resourceIdentifier"`
+
+	// FeedbackRules defines what resource status field should be returned.
+	// +kubebuilder:validation:Required
+	// +required
+	FeedbackRules []FeedbackRule `json:"feedbackRule"`
```

Comment:
s/feedbackRule/feedbackRules/

Reply:
updated, I actually change this name to be manifestConfigs. I am thinking we might have more config option based on resource identifier later.